### PR TITLE
Clarify meaning of -a dns-plugin

### PR DESCRIPTION
**Pull request by bmw**
Inspired by the comment at https://community.letsencrypt.org/t/how-to-create-certificates-for-internal-websites/90377/22.

### CODE REVIEW

File: _scripts/instruction-widget/templates/getting-started/apache.html [hunk 1]
```diff
@@ -42,9 +42,11 @@ <h4>Note:</h4>
 <pre>$ sudo {{base_command}} -a dns-plugin -i apache -d "*.example.com" -d example.com --server https://acme-v02.api.letsencrypt.org/directory</pre>
 <p>
 You'll need to replace <tt>dns-plugin</tt> with the name of the DNS plugin you
-want to use. You may also need to provide additional flags such as the path to
-your API credentials as described in the documentation for the DNS plugin
-linked above.
+want to use. For instance, to use the Cloudflare plugin, you would use
+<tt>dns-cloudflare</tt>. You may also need to provide additional flags such as
+the path to your API credentials. To learn more about how to use these plugins,
+see the documentation for DNS plugin you are trying to use found in the link
```
Comment (adferrand) [inline _scripts/instruction-widget/templates/getting-started/apache.html]: for the DNS plugin

File: _scripts/instruction-widget/templates/getting-started/nginx.html [hunk 2]
```diff
@@ -39,10 +39,12 @@ <h4>Note:</h4>
...
 <p>
-You'll need to replace <tt>dns-plugin</tt> with the name of the DNS plugin
-you want to use. You may also need to provide additional flags such as the path
-your API credentials as described in the documentation for the DNS plugin
-linked above.
+You'll need to replace <tt>dns-plugin</tt> with the name of the DNS plugin you
+want to use. For instance, to use the Cloudflare plugin, you would use
+<tt>dns-cloudflare</tt>. You may also need to provide additional flags such as
+the path to your API credentials. To learn more about how to use these plugins,
+see the documentation for DNS plugin you are trying to use found in the link
```
Comment (adferrand) [inline _scripts/instruction-widget/templates/getting-started/nginx.html]: for the DNS plugin

File: _scripts/instruction-widget/templates/getting-started/apache.html (hunk 1 above)
Comment (bmw) [reply _scripts/instruction-widget/templates/getting-started/apache.html]: Thanks for correcting my English :)

File: _scripts/instruction-widget/templates/getting-started/nginx.html (hunk 2 above)
Comment (bmw) [reply _scripts/instruction-widget/templates/getting-started/nginx.html]: Thanks for correcting my English :)

**Pull request merged**